wrangling.Rmd

```diff
@@ -619,56 +602,56 @@ column_range
+`contains("_")` as the second argument in the `select` function, since we notice
+the columns we want contain underscores and the others don't. \index{select!contains}
 
 ``` {r 02-select-helper-contains}
 select(tidy_lang, contains("_"))
 ```
 
-There are many different `select` helpers, which are very handy for selecting
+There are many different `select` helpers that select
 variables based on certain criteria. 
-Check out the Additional Resources section at the end of the chapter 
-for a comprehensive resource on `select` helpers.
+The additional resources section at the end of this chapter 
+provides a comprehensive resource on `select` helpers.
 
 ## Using `filter` to extract rows
 
-### Using `filter` to extract rows with `==`
+Next, we revisit the `filter` function from Chapter \@ref(intro), 
+which lets us create a subset of rows from a data frame. 
+Recall the two main arguments to the `filter` function:
+the first is the name of the data frame object, and \index{filter!logical statements}
+the second is a *logical statement* to use when filtering the rows.
+`filter` works by returning the rows  where the logical statement evaluates to `TRUE`.
+This section will highlight more advanced usage of the `filter` function.
+In particular, this section provides an in-depth treatment of the variety of logical statements
+one can use in the `filter` function to select subsets of rows.
 
-Recall we can use the `filter` function to obtain the subset of rows with
-desired values from a data frame. Again, our first argument is the name of the
-data frame object.
-Here we will continue to work with the `tidy_lang` data frame 
-to demonstrate more details on how filter works.
-The second argument to `filter` 
-is a logical statement to use when filtering the rows. 
-`filter` works by returning the rows 
-where the logical statement evaluates to `TRUE`.
-Suppose we are only interested in subsetting the rows for the
-official languages of Canada (English and French) from this data set. 
-We can do this by using the *equivalency operator* (`==`) 
+### Using `filter` to extract rows with `==`
+Suppose we are only interested in the subset of rows in `tidy_lang` corresponding to the
+official languages of Canada (English and French).
+We can `filter` for these rows by using the *equivalency operator* (`==`) 
 to compare the values of the `category` column 
 with the value `"Official languages"`. 
 With these arguments, `filter` returns a data frame with all the columns 
 of the input data frame 
-but only the rows we asked for in our logical filter statement - 
+but only the rows we asked for in our logical filter statement, i.e., 
 those where the `category` column holds the value `""Official languages"`.
 We name this data frame `official_langs` and will primarily focus on this 
-data frame for the remainder of this chapter (except for the next example).
+data frame for the remainder of this chapter.
```

Comment:
I don't know if we use this data for the rest of the chapter.. I think its used for the next few sections but not the rest of the chapter e.g. 3.8.1 we use the tidy_lang again

Reply:
fixed